cookbooks/fail2ban/attributes/default.rb

```diff
@@ -0,0 +1,265 @@
+# fail2ban.conf
+#          1 = ERROR
+#          2 = WARN
+#          3 = INFO
+#          4 = DEBUG
+default['fail2ban']['loglevel'] = 'DEBUG'
+default['fail2ban']['socket'] = '/var/run/fail2ban/fail2ban.sock'
+default['fail2ban']['logtarget'] = '/var/log/fail2ban.log'
+default['fail2ban']['pidfile'] = '/var/run/fail2ban/fail2ban.pid'
+
+# global config
+# Available versions for v6 are:
+# - 0.10.2-2
+default['fail2ban']['version'] = '0.10.2-2'
```

Comment:
We are residing to what Ubuntu offers in regards to package version these days. Please remove any hardcoded package version on recipe(s)